### ISSUE TITLE
Example command from README.md fails

### ISSUE THREAD

**Issue by maxamillion** [DBuildService/atomic-reactor#205]
When attempting to follow the instructions in the `README.md` the command to perform a build fails.

```
$ atomic-reactor build --method hostdocker --build-image buildroot --image test-image --git-url "https://github.com/TomasTomecek/docker-hello-world.git"
usage: atomic-reactor [OPTIONS] build
atomic-reactor build: error: argument source__provider: invalid choice: 'hostdocker' (choose from 'git', 'path', 'json')

```

**Comment by TomasTomecek** [comment]
Thanks for bugreport! Over the past few weeks we have changed reactor's interface and right now we are in state of updating docs to reflect the changes. Will take care of this.

**Comment by bkabrda** [comment]
Yeah, I noted that we need to fix this in https://github.com/DBuildService/atomic-reactor/issues/48#issuecomment-109168847, but then never really got to it. I'll try fixing it today.

**Comment by bkabrda** [comment]
Fixed in #208
